### ISSUE TITLE
[ModuleManager] when it's set to Active:no - page fails to load 

### ISSUE THREAD

**Issue by christinerogers** [aces/Loris#9245]
Basically this module should never be configured to Active:No -- but it should still fail a bit more elegantly here on the off chance it is. 

currently it loads with a blank page saying "this module could not be found" if it's set to Active:no (in the `modules` table i think)

found during C-BIg testing may 9.  filed a bug after discussion with @ridz1208  and Dave.

**Comment by driusan** [comment]
The problem isn't that it fails when Active='N', the problem is that it fails when it's not on the file system (even if it's inactive)

**Comment by ridz1208** [comment]
@driusan maybe it can be a try catch to get the module info and if it fails it can just replace with NA or some other message

**Comment by christinerogers** [comment]
still seems like an important bug (did not test)

**Comment by sruthymathew123** [comment]
@driusan @christinerogers I agree that this module shouldn't be set No; But I am getting the below attached attached message when set to 'No'..

![Image](https://github.com/user-attachments/assets/5c123ac1-f918-474b-b311-5a0fa3ce2b22)

What is the fix we need to implement here. I am little confused.

**Comment by charliehenrib** [comment]
Could probably be fixed if the changes in https://github.com/aces/Loris/issues/7083 were implemented.
